SPECS/python-platformdirs/python-platformdirs.spec

```diff
@@ -0,0 +1,61 @@
+%global srcname platformdirs
+%global common_description %{expand:
+A small Python module for determining appropriate platform-specific dirs, e.g.
+a "user data dir".}
+Summary:        Python module for determining appropriate platform-specific dirs
+Name:           python-%{srcname}
+Version:        2.0.0
+Release:        2%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/platformdirs/platformdirs
+Source0:        %{url}/archive/%{version}/%{srcname}-%{version}.tar.gz
```

Comment:
`Source0` written this way will make `rpmbuild` look for `%{srcname}-%{version}.tar.gz`, which doesn't match the file mentioned in the signatures JSON. Please make sure all of these file names align.